homeassistant/components/nuki/lock.py

```diff
@@ -153,6 +120,11 @@ def is_locked(self):
         """Return true if lock is locked."""
         return self._nuki_device.is_locked
 
+    @property
+    def is_door_sensor_activated(self):
```

Comment:
Where is this used?

Reply:
This is used in the binary_sensor: https://github.com/home-assistant/core/blob/899f8f1d74681d533b5247b42b77e3d5f8f1c2c1/homeassistant/components/nuki/binary_sensor.py#L30

Reply:
I think I mixed this up a bit, in the binary_sensor it's using the `is_door_sensor_activated` from the nuki library and not from this entity. I will remove this then.